docs/node-operators/requirements/data-node-security.md

```diff
@@ -0,0 +1,264 @@
+---
+sidebar_position: 3
+title: Running a secure data node
+sidebar_label: Data node security
+hide_title: false
+---
+
+Before reading this page, we recommend reading the [validator node security page](./security.md) for useful security guidance. The following instructions describe how to set up a secure data node with exposed APIs.
+
+As a node operator, you must provide TLS to the APIs exposed to the internet to provide secure communication between users and your server. GraphQL subscriptions won't work without TLS. 
+
+To setup a secure server, you have a few options:
+
+- The data node has an internal mechanism to provide the TLS and auto-request for the certificate.
+- A proxy service like `nginx` or `caddy` if you wish to pass requests with a DMZ.
+
+The data node exposes 3 APIs with two separate ports:
+
+- gRPC port: Default value is `3007` - used for gRPC API
+- Gateway port: Default value is `3008` - used for the GraphQL and REST APIs.
+
+You should expose both gRPC and Gateway with a TLS. However, the data node's internal mechanism allows you to expose only the Gateway port.
+
+## List of secure data nodes
+
+The Vega consensus validators run publicly available data nodes. See a list of publicly available data nodes in the [networks GitHub repository ↗](https://github.com/vegaprotocol/networks/blob/master/mainnet1/mainnet1.toml). All of the listed (not commented) data nodes have TLS support.
+
+## Run a secure data node
+
+### A bit about TLS
+
+The below examples show the autogenerated certificate from "Lets Encrypt", but you can also buy a different certificate or use the certificate you already own. Just be sure to provide paths to the certificate and key in your configuration.
+
+If you decide to use the certbot to obtain the certficate please read the below documentation:
+
+- [Install certbot ↗](https://www.inmotionhosting.com/support/website/ssl/lets-encrypt-ssl-ubuntu-with-certbot/)
+- [Best practices for setting a cron job for Let's Encrypt (Certbot) renewal ↗](https://serverfault.com/questions/790772/best-practices-for-setting-a-cron-job-for-lets-encrypt-certbot-renewal)
+
+### Use internal mechanism to expose Gateway
+
+Follow on to use the internal data node mechanism to expose Gateway (GraphQL and REST APIs) with TLS.
+
+Assumptions:
+
+- Data node service is configured and running, or ready to start.
+- You have the spare domain pointing to your server.
+- You have free 443 port on your server where the data node is running.
+- Your server is publicly available on the internet
+
+Limitations:
+
+Unfortunately, the data node cannot set up TLS for the GRPC port at the moment. It can be done only for the Gateway port(GraphQL and the REST APIs).
```

Comment:
```suggestion
Unfortunately, the data node cannot set up TLS for the gRPC port. It can be done only for the Gateway port (GraphQL and the REST APIs).
```